app/views/docs/rest.phtml

```diff
@@ -1,12 +1,337 @@
+
+<div class="ide margin-bottom" data-lang="http" data-lang-label="HTTP">
+    <pre class="line-numbers"><code class="prism language-http" data-prism>GET /v1/databases/{databaseId}/collections/{collectionId}/documents  HTTP/1.1
+Host: [HOSTNAME_OR_IP]
+Content-Type: application/json
+X-Appwrite-Project: [PROJECT_ID]
+X-Appwrite-Key: [API_KEY]
+X-Appwrite-Response-Format: 1.0.0</code></pre>
+</div>
+
+<h2>File Handling</h2>
+<p>Appwrite implements resumable, chunked uploads for files larger than 5MB. Chunked uploads send files in chunks of 5MB to reduce memory footprint and increase resilience when handling large files. Appwrite SDKs will automatically handle chunked uploads but it is possible to implement this with the REST API directly.</p>
+
+<p>Upload endpoints in Appwrite such as Create File and Create Deployment are different from other endpoints. These endpoints take multipart form data instead of JSON data. To implement chunked uploads, you will need to implement the following headers:</p>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 300px">Header</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>X-Appwrite-Project: [PROJECT-ID]</code></td>
+        <td>Required, contains the ID of your Appwrite Project to the REST API.</td>
+    </tr>
+    <tr>
+        <td><code>Content-type: multipart/form-data; boundary=[FORM-BOUNDRY]</code></td>
+        <td>Required, contains the content type of the HTTP request and provides a <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/POST">boundry</a> which is used to parse the form data.</td>
+    </tr>
+    <tr>
+        <td><code>content-range: bytes [BYTE-RANGE]</code></td>
+        <td>Required, contains information about which bytes are being transmitted in the format <code>[FIRST-BYTE]-[LAST-BYTE]/[TOTAL-BYTES]</code>.</td>
+    </tr>
+    <tr>
+        <td><code>x-appwrite-id: [FILE-ID]</code></td>
+        <td>Required, contains which file this chunk belongs to.</td>
+    </tr>
+    <tr>
+        <td><code>X-Appwrite-key: [API-KEY]</code></td>
+        <td>Optional, used for authentication in server integrations. <b>Do not use API keys in client applications.</b></td>
+    </tr>
+    </tbody>
+</table>
+
+<p>The multipart form data is structured as follows:</p>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 100px">Key</td>
+        <td style="width: 150px">Value</td>
+        <td style="width: 100px">File Name</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>fileId</td>
+        <td><code>[FILE-ID]</code></td>
+        <td>n/a</td>
+        <td>Required, contains the file ID of the new file. Use <code>"unique()"</code> for a UUID.</td>
```

Comment:
Let's use  `required` and `optional` as a pill like we have here https://github.com/appwrite/docs/blob/c5660a58a5f201b930af3070f27434e0597b9d21/app/views/docs/graphql.phtml#L23-L25